test/unit/plugins/parser/divclass.test.js

```diff
@@ -124,7 +124,7 @@ describe("divclass", () => {
     it("works without content - but only if separated by FOUR newlines", () => {
       const input = "[empty]\n\n\n\n[/empty]";
       const output = processor.sourceToRedacted(input);
-      expect(output).toEqual("[][0]\n\n[/][0]\n");
+      expect(output).toEqual("[][0]\n\n\n\n[/][0]\n");
```

Comment:
Why does this add extra new lines?

Reply:
I tweaked the way block redactions get rendered to make the code simpler, which had this as a side effect